pipelines/managed/rh-advisories/rh-advisories.yaml

```diff
@@ -283,7 +283,6 @@ spec:
       runAfter:
         - apply-mapping
     - name: populate-release-notes
-      timeout: "4h"  # temp workaround until github.com/konflux-ci/release-service/issues/603 is fixed
```

Comment:
So this means all the tasks will have a timeout of 1 hour now, right? And there is no way users can increase this. I wonder if that's gonna be enough. For example, just recently we got this ticket where the user was saying `rh-sign-image-cosign` took 1 hour 4 minutes 32 seconds with 60 components.

I am not against this change, but I think it's likely we will need to increase it later.